zoe-cli/src/main.kt

```diff
@@ -40,6 +64,10 @@ fun <T> withZoe(customizeContext: Context.Builder.() -> Unit = {}, action: (ZoeC
                 )
                 .context(customizeContext)
 
+        val modules = listOf(mainModule { command.buildCliContext() }) + additionalModules
```

Comment:
here it is more clear now in which order the koin modules are defined and we are able to pass additional modules. In this particular case: the test module which overrides the EnvConfig definition